kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -2748,6 +2748,33 @@ def remove_vm_state_from_cache(uuid):
+        return
+
+    if last_libvirtd_pid_for_vm_priority_sync == current_pid:
+        return
+
+    for vm_uuid, state in list(states.items()):
+        if state != Vm.VM_STATE_RUNNING:
+            continue
+
+        try:
+            linux.sync_vm_live_cpu_shares_from_config(vm_uuid)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)